docker/test/integration/steps/steps.py

```diff
@@ -0,0 +1,243 @@
+from MiNiFi_integration_test_driver import MiNiFi_integration_test
```

Comment:
Could these steps possible be split up into multiple files corresponding to each test suite?

Reply:
Yes, but it makes the directory structure worse, as each feature would need to live in its own subdirectory as described here:
https://stackoverflow.com/questions/36613050/modular-structure-for-behave-tests
If we need further structural changes, let's do it in another PR.

Reply:
I see, thanks for the info.